inbc-program/README.md

```diff
@@ -114,11 +122,21 @@ inbc sota
      --releasedate 2022-02-22 
      --username <username>
 ```
-#### Edge Device on Ubuntu
+#### Edge Device on Ubuntu in Update/Full mode
 ```
 inbc sota
 ```
 
+#### Edge Device on Ubuntu in download-only mode
+```
+inbc sota --mode download-only
+```
+
+#### Edge Device on Ubuntu in no-download mode
+```
+inbc sota --mode no-download
```

Comment:
should it be `inbc sota --command update --mode no-download`?

Reply:
@yengliong93 by default --command is update so don't have to use it is what Natalie suggested

These are examples, so you don't want to use the brackets. And since there are default values set, you won't need to show the command and mode unless it's different than the default.

inbc sota (don't need the update and full because they are the updates)
inbc sota --mode download-only
inbc sota --mode no-download